angular.json

```diff
@@ -132,7 +132,9 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "kendraio-app:build"
+            "browserTarget": "kendraio-app:build",
+            "host": "0.0.0.0",
+            "disableHostCheck": true
```

Comment:
This is another change that is required to work with remote containers on dynamic urls. 

I've not been able to find much discussion around these changes. They all introduce potential issues. 
However, most of the issues result in access to source code, which is not a problem for us. 

They also could potentially reveal secrets stored on a server... but since we store all data in the client, this does not impact us either. 

So, I don't think we need to worry. 


We will however get warnings when we start a dev server.